coriolisclient/cli/migrations.py

```diff
@@ -158,6 +160,10 @@ def get_parser(self, prog_name):
                             help='JSON mapping between identifiers of '
                             'networks on the source and identifiers of '
                             'networks on the destination.')
+        parser.add_argument('--source-environment', default={},
```

Comment:
If you insist on a default, it should be the string `"{}"`, right?
IMO, this should default to `None` and not be sent int he request at all if not provided. (it's not like the dest-env which we know for a fact will always be required)